Automattic/jetpack
# CLI: Let auto-changelogger work after commits

#

#### Changes proposed in this Pull Request:

This PR lets the auto changelog prompter to work after you've committed, instead of pre-commit, which wasn't very usefull.

#### Jetpack product discussion


N/a
#### Does this pull request change what data or activity we track or use?


no
#### Testing instructions:






* Make a change to a project and commit it
* Run `jetpack changelog add` - it should find the project that was touched and prompt you to add a changelog for it.
* Make changes to multiple projects and run `jetpack changelog add` - it should find all the projects that need one and let you add a single changelog file for all.

There are probably some use case gaps and quirks, and the file itself is kinda rough and needs some refactoring, but looking for a quick win v1 that's more useful than the pre-commit run now.
